keras_cv/layers/__init__.py

```diff
@@ -73,3 +73,6 @@
 from keras_cv.layers.regularization.squeeze_excite import SqueezeAndExcite2D
 from keras_cv.layers.regularization.stochastic_depth import StochasticDepth
 from keras_cv.layers.spatial_pyramid import SpatialPyramidPooling
+from keras_cv.layers.transformer_encoder import TransformerEncoder
+from keras_cv.layers.vit_layers import PatchEmbedding
+from keras_cv.layers.vit_layers import Patching
```

Comment:
Also make Patching and PatchEmbeddings private layers

Reply:
Just removing them from the __init__ file is enough, right?

Reply:
yes

Reply:
Done

Reply:
Removed in the latest commit (accidentally left them in in the one before for testing on a local Jupyter Notebook)